pkgs/development/python-modules/daff/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "daff";
+  version = "1.3.46";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-ItDan9ajJ1tUySapyXsYD5JYqtZRE+oY8/7FLLrc2Bg=";
+  };
+
+  propagatedBuildInputs = [
+  ];
+
```

Comment:
We want to execute the tests if possible https://github.com/paulfitz/daff/tree/master/test

Reply:
I believe these tests are not related to the python package `daff`. They are for the companion JS package.